src/Castle.Core.Tests/MultipleRepeatedInterfaceTestCase.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+using Castle.DynamicProxy;
+using NUnit.Framework;
+
+namespace CastleTests
+{
+    [TestFixture]
+    public class MultipleRepeatedInterface

```

Comment:
Use the `BasePEVerifyTestCase` base class for your tests. Have a look at some other test cases how they use it.